### PR TITLE
Added new methods

### PR DESCRIPTION
**Pull request by gfbenatto**
Added POST, PUT and DELETE methods and field 'country' into the dictionary.

### CODE REVIEW

File: store/app.py [hunk 1]
```diff
@@ -1,10 +1,74 @@
...
+@app.route('/api/v1/product/<int:product_id>')
+def get_product_id(product_id):
+    for product in products:
+        if product['id'] == product_id:
+            return jsonify(products[product_id - 1]), 200
+    return jsonify({'Error':'Not Found'}), 404
+
+
+@app.route('/api/v1/product/<int:product_id>', methods=['PUT'])
+def change_country(product_id):
```
Comment (patito) [inline store/app.py]: the name of the function is `change_country` and the parameter is `product_id`? weird right?

File: store/app.py [hunk 2]
```diff
@@ -1,10 +1,74 @@
...
+            'id': 3,
+            'size': 9,
+            'price': 45,
+            'country': 'EUA'
+        }
+]
+
+@app.route('/')
+@app.route('/api/v1/products')
+def home():
```
Comment (patito) [inline store/app.py]: why the name of this function is home?

File: store/app.py [hunk 3]
```diff
@@ -1,10 +1,74 @@
...
+    return jsonify(products)
+
+
+@app.route('/api/v1/product/<int:product_id>')
+def get_product_id(product_id):
```
Comment (patito) [inline store/app.py]: i think `get_product_by_id` is a better name.

File: store/app.py [hunk 4]
```diff
@@ -1,10 +1,74 @@
...
+    data = request.get_json()
+    products.append(data)
+    return jsonify(data), 201
+
+
+@app.route('/api/v1/product/<int:product_id>', methods=['DELETE'])
+def remove_item(product_id):
+    for product in products:
+        if product['id'] == product_id:
+            del products[product['id'] - 1]
```
Comment (patito) [inline store/app.py]: I think this is a bad approach: `del products[product['id'] - 1]`
You could try something like:
```
for counter, product in enumerate(products):
    ...
```

**Review by patito — CHANGES_REQUESTED** [review]
I have made some suggestions

**Comment by gfbenatto** [comment]
Ok, I am checking.

**Review by patito — APPROVED** [review]

**Pull request merged**